src/Workspaces/Core/Portable/FindSymbols/FindReferences/BaseTypeFinder.cs

```diff
@@ -43,29 +39,39 @@ internal static partial class BaseTypeFinder
 
                     if (sourceMember.Symbol != null)
                     {
-                        // These are explicit interface implementations matching by name.
-                        if (type?.TypeKind == TypeKind.Interface)
-                        {
-                            if (symbol.ContainingType?.TypeKind == TypeKind.Class || symbol.ContainingType?.TypeKind == TypeKind.Struct)
-                            {
-                                var implementation = symbol.ContainingType.FindImplementations(sourceMember.Symbol, solution.Workspace);
-
-                                if (implementation != null &&
-                                    SymbolEquivalenceComparer.Instance.Equals(implementation.OriginalDefinition, symbol.OriginalDefinition))
-                                {
-                                    results.Add(sourceMember);
-                                }
-                            }
-                        }
-                        else if (SymbolFinder.IsOverride(solution, symbol, sourceMember.Symbol, cancellationToken))
+                        // Add to results overridden members only. Do not add hidden members.
```

Comment:
i don' understand the "hidden members" comment.

Reply:
new and Shadows

---
In reply to: [313678543](https://github.com/dotnet/roslyn/pull/37942#discussion_r313678543) [](ancestors = 313678543)

Reply:
Interface I
 Sub M()
End Interface

Class C
Implements I
Public Sub N() Implements I.M()
End Sub
End Class

Class D
Inherits C
Public Shadows Sub N() 
End Sub
End Class

We need to add C.N() to the list to find its implementation of I.M() because D.N() does not have I.M() in the list of its implementations.

---
In reply to: [313678543](https://github.com/dotnet/roslyn/pull/37942#discussion_r313678543) [](ancestors = 313678543)